prime-router/src/main/kotlin/Submission.kt

```diff
@@ -56,6 +57,18 @@ class DetailedSubmissionHistory(
     val errorCount: Int
         get() = errors.size
 
+    /**
+     * Number of destinations.
+     */
+    val destinationCount: Int
+        get() = destinations.size
+
+    /**
+     * Number of report items.
+     */
+    val reportItemCount: Int?
+        get() = actionResponse?.reportItemCount
```

Comment:
Could start getting away from using the `actionResponse` now and move to using the information in the stored report.

Reply:
there should be a single report_file related to the `Receive` action that has the total number of originally submitted items in it.